### PR TITLE
32 api

### PR DESCRIPTION
**Pull request by Cranekick50**
#32 resolved - data displays on page but needs to be customized.

### CODE REVIEW

**Review by juanlajara — APPROVED** [review]
Sweet

**Review by marlon20m — APPROVED** [review]
Looks great !

**Review by joshuabogart — APPROVED** [review]
nice work on the APIs dude!

**Comment by AshleyLerma** [comment]
What is causing these errors?
     - js:178 GET http://localhost:3000/api/search/margarita 500 (Internal Server Error)
     - Uncaught (in promise) Error: Request failed with status code 500
    at createError (createError.js:16)
    at settle (settle.js:17)
    at XMLHttpRequest.handleLoad (xhr.js:61)
- The Search component needs to be put into a folder for consistent organization
- Is the loading icon not showing for anyone else?

File: client/src/components/Search.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from "react";
+import API from "../utils/api";
+
+function Search() {
+  const drink = "margarita";
```
Comment (AshleyLerma) [inline client/src/components/Search.js:5]: Will this be replaced to equal the value of a search bar or something along those lines?

**Review by AshleyLerma — APPROVED** [review]

**Pull request merged**